"SSH key management" page - Dead link update (#288)
CleverCloud/documentation: content/doc/applications/docker/_index.md

{{% content/more-config %}}

{{% content/url_healthcheck %}}

### Commonly encountered issues

{{% details title="I can't reach my app port" closed="true" %}}

> You probably want to use a specific port for their apps. However, you can't directly reach for that port, as your request will go through Clever Cloud's reverse proxies, then through the VM.

Reviewer: who's  "them" in `their apps` ?

Here are the steps:

> * Create a TCP redirection that will allow entry through our load balancers, that will come out as 4040 or 8080 on the Docker. You can find a guide here:

Reviewer: `coming out on the Docker` isn't clear enough. Which workflow are you trying to describe here?

```suggestion
* Create a TCP redirection to allow entry through Clever Cloud load balancers, with port 4040 or 8080 coming out on the Docker. Check [this guide](/doc/administrate/tcp-redirections) for more info.
```

> <https://developers.clever-cloud.com/doc/administrate/tcp-redirections/>

Reviewer: ```suggestion

```

If you are not using 4040/8080 as your app port, you must redirect those to the desired port of your app.  

> * Set environnement variable `CC_DOCKER_EXPOSED_TCP_PORT` or `CC_DOCKER_EXPOSED_HTTP_PORT` (depending on technology) to translate 4040/8080 to anyport your wish on your app.

Reviewer: ```suggestion
* Set environment variable `CC_DOCKER_EXPOSED_TCP_PORT` or `CC_DOCKER_EXPOSED_HTTP_PORT` (depending on technology) to translate 4040/8080 to any port of your choice on your app.
```

> You can connect to your app on your custom port by connecting to it from the outside on the TCP port that has been provided. 

Reviewer: Unclear sentence, needs to be more specific : what is connecting from where ? Maybe rephrase it like this for now:

 ```suggestion
You can connect to your app on your custom port on the provided TCP port. 
```
